ersilia/publish/test.py

```diff
@@ -379,24 +372,32 @@ def check_example_input(self, output):
     it is not consistent, an InconsistentOutput error is raised. Lastly, it makes sure that the number of 
     outputs equals the number of inputs.  
     """
-
+    
+    
+    
     @throw_ersilia_exception
     def check_consistent_output(self):
-        # self.logger.debug("Confirming model produces consistent output...")
+        def compute_mrae(values1, values2):
```

Comment:
I'm a bit confused, are you trying to calculate Mean absolute error or Relative absolute error? Could you point me to the formula that you are using? Also why not use a standard implementation within numpy/scipy?

Reply:
The Mean Relative Absolute Error calculated the mean of the **relative** **absolute** **error**, so I am calculating the relative error between each pair of values and then I take the average of these relative errors across all data points. There did exist standard implementations, but I was running into divide by 0 errors.
<img width="286" alt="Screenshot 2024-08-12 at 2 07 35 PM" src="https://github.com/user-attachments/assets/ead7b755-9a80-49ac-b8ee-c870b20b9fb3">

However, after testing this implementation on different models, I found that there are high inflated error values that cause the test to fail when it should pass (similar to #1221). So, I am switching to the root mean square error to solve this issue.